### PR TITLE
Updates pages in the CMS documentation for packages across Umbraco versions.

### PR DESCRIPTION
**Pull request by AndyButland**
## Description

As requested by the docs team, this PR updates pages in the CMS documentation for packages across Umbraco versions.

- Updated the section on "value connectors" in the Deploy documentation, removed the partly out of date details on this topic from the CMS packages section and linked to the Deploy docs.
- Copied over the already updated "maintaining packages" section for Umbraco 10 into 12, 13 and 14

## Type of suggestion

* [ ] Typo/grammar fix
* [X] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

CMS and Deploy

## Deadline (if relevant)

Can be reviewed and published at any time.

### CODE REVIEW

**Comment by eshanrnh** [comment]
Thanks for the PR, @AndyButland 🙌 We will review it as soon as we can 💪

File: 10/umbraco-cms/extending/packages/packages-on-umbraco-cloud.md [hunk 1]
```diff
@@ -38,277 +38,10 @@ You may sometimes choose to save data in a file. Could be a separate config file
...
+A value connector is an extension to Umbraco Deploy that allows you to transform data when you deploy content of any kind between environments. When it comes to packages, one reason you need to consider these if you are supporting deploying content properties that rely on integer IDs. Content and other Umbraco data has two identifiers - an integer and a GUID. The GUID is consistent between environments but the integer ID is not. As such, if transferring content between environments and relying on integer IDs, you'll need to include a value connector to transform the value.
 
-An example of creating one for your package would be if you had a custom property editor that allowed you to write in the ID of a media node. Not a very usable property editor but it will work for this example.
+They also manage dependencies for property data. That means that if you save for example an ID of an image in your property editor, then you can make sure that not only the ID but the actual image is transferred as well.
```
Comment (jonat123) [inline 10/umbraco-cms/extending/packages/packages-on-umbraco-cloud.md:45]:
```suggestion
They also manage dependencies for property data. If you save an ID of an image in your property editor, you can make sure the ID and the image are transferred as well.
```

File: 12/umbraco-cms/extending/packages/packages-on-umbraco-cloud.md [hunk 2]
```diff
@@ -34,275 +34,10 @@ You may sometimes choose to save data in a file. Could be a separate config file
...
+A value connector is an extension to Umbraco Deploy that allows you to transform data when you deploy content of any kind between environments. When it comes to packages, one reason you need to consider these if you are supporting deploying content properties that rely on integer IDs. Content and other Umbraco data has two identifiers - an integer and a GUID. The GUID is consistent between environments but the integer ID is not. As such, if transferring content between environments and relying on integer IDs, you'll need to include a value connector to transform the value.
 
-An example of creating one for your package would be if you had a custom property editor that allowed you to write in the ID of a media node. Not a very usable property editor but it will work for this example.
+They also manage dependencies for property data. That means that if you save for example an ID of an image in your property editor, then you can make sure that not only the ID but the actual image is transferred as well.
```
Comment (jonat123) [inline 12/umbraco-cms/extending/packages/packages-on-umbraco-cloud.md:41]:
```suggestion
They also manage dependencies for property data. If you save an ID of an image in your property editor, you can make sure the ID and the image are transferred as well
```

File: 13/umbraco-cms/extending/packages/packages-on-umbraco-cloud.md [hunk 3]
```diff
@@ -34,275 +34,10 @@ You may sometimes choose to save data in a file. Could be a separate config file
...
+A value connector is an extension to Umbraco Deploy that allows you to transform data when you deploy content of any kind between environments. When it comes to packages, one reason you need to consider these if you are supporting deploying content properties that rely on integer IDs. Content and other Umbraco data has two identifiers - an integer and a GUID. The GUID is consistent between environments but the integer ID is not. As such, if transferring content between environments and relying on integer IDs, you'll need to include a value connector to transform the value.
 
-An example of creating one for your package would be if you had a custom property editor that allowed you to write in the ID of a media node. Not a very usable property editor but it will work for this example.
+They also manage dependencies for property data. That means that if you save for example an ID of an image in your property editor, then you can make sure that not only the ID but the actual image is transferred as well.
```
Comment (jonat123) [inline 13/umbraco-cms/extending/packages/packages-on-umbraco-cloud.md:41]:
```suggestion
They also manage dependencies for property data. If you save an ID of an image in your property editor, you can make sure the ID and the image are transferred as well
```

**Review by jonat123 — CHANGES_REQUESTED** [review]
@AndyButland Looks good :-) 

I have made a couple of suggestions to shorten some sentences.

Let me know what you think! 😄

File: 14/umbraco-cms/extending/packages/packages-on-umbraco-cloud.md [hunk 4]
```diff
@@ -34,280 +34,11 @@ You may sometimes choose to save data in a file. Could be a separate config file
...
+A value connector is an extension to Umbraco Deploy that allows you to transform data when you deploy content of any kind between environments. When it comes to packages, one reason you need to consider these if you are supporting deploying content properties that rely on integer IDs. Content and other Umbraco data has two identifiers - an integer and a GUID. The GUID is consistent between environments but the integer ID is not. As such, if transferring content between environments and relying on integer IDs, you'll need to include a value connector to transform the value.
 
-An example of creating one for your package would be if you had a custom property editor that allowed you to write in the ID of a media node. Not a very usable property editor but it will work for this example.
+They also manage dependencies for property data. That means that if you save for example an ID of an image in your property editor, then you can make sure that not only the ID but the actual image is transferred as well.
```
[the remark this answers is not in the document]
Comment (AndyButland) [reply 14/umbraco-cms/extending/packages/packages-on-umbraco-cloud.md]: Didn't think this was quite right, but I've pushed an update that gets under the 25 word rule.

**Pull request merged**